operators/deploy/tenant-operator/templates/deployment.yaml

```diff
@@ -47,6 +47,10 @@ spec:
             - "--webhook-bypass-groups={{ .Values.webhook.deployment.webhookBypassGroups }}"
             - "--sandbox-cluster-role={{ .Values.configurations.sandboxClusterRole }}"
             - "--max-concurrent-reconciles={{ .Values.configurations.maxConcurrentReconciles }}"
+            - "--user-pvc-size-bytes={{ .Values.configurations.userPvcSizeBytes }}"
+            - "--storage-class-name={{ .Values.configurations.storageClassName }}"
+            - "--rook-ceph-namespace={{ .Values.configurations.rookCephNamespace }}"
+            - "--mydrive-pvcs-namespace={{ .Values.configurations.mydrivePvcsNamespace }}"
```

Comment:
I would uniform the names of this parameters (both the args and in Helm), so that they all start with mydrive-pvcs-.